app/main.py

```diff
@@ -1,3 +1,3 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
-    pass
+
+    return phrase.lower().count(letter.lower())
```

Comment:
The return statement is currently returning a float because the `count` method returns an integer, but the division operator `/` is used, which results in a float. To fix this, you should remove the division operator and directly return the result of `phrase.lower().count(letter.lower())` to ensure the function returns an integer as specified in the function signature.